opslevel/resource_opslevel_team_tag.go

```diff
@@ -63,7 +65,7 @@ func (teamTagResource *TeamTagResource) Schema(ctx context.Context, req resource
 		MarkdownDescription: "Team Tag Resource",
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
-				Description: "The ID of this resource.",
+				Description: "The ID of this resource. Formatted as <team-id>:<tag-id>.",
```

Comment:
I'm confused why we have to change this?  Can the `ImportState` function not do the splitting logic and then find the tag by ID and store this as the ID without this format?

Additionally `ImportState` should beable to do all the stuff it needs to.  I think you are conflating the `ImportStatePassthroughID` - we have to write our own importer function and not rely upon reusing the `Read` function.